<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>findCat</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/m.jpg</normaloff>images/m.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/图标3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>浏览/查询猫猫信息</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_cat">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>721</width>
      <height>209</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>昵称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>颜色</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>年龄</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>性格</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>种类</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>常出现的位置</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_allcat">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>430</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>显示全部猫猫</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_catname">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>请输入猫猫昵称</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_findcat">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/猫6.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/猫7.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
